src/matrix/sprig-matrix.ts

```diff
@@ -112,6 +112,43 @@ export function resetTempMatrixBuffer() {
   }
 }
 
+export function isTmpVec(v: Float32Array): boolean {
+  return v.buffer === buffer;
+}
+
+// TODO(@darzu): generalize and put in util.ts?
+export function findAnyTmpVec(
```

Comment:
new util function that can search through any* JS value to find temp vecs and returns the path to that vec